### ISSUE TITLE
Doesn't work without symbiote/silverstripe-queuedjobs installed

### ISSUE THREAD

**Issue by matthewrayner** [brettt89/silverstripe-garbage-collector#12]
Throws error without symbiote/silverstripe-queuedjobs module installed.

**Fatal error: Class 'Symbiote\QueuedJobs\Services\AbstractQueuedJob' not found**

Is this module supposed to be required?

**Comment by michalkleiner** [comment]
Should only throw when trying to run the `GarbageCollectorJob`. What action are you performing when it fails?

**Comment by matthewrayner** [comment]
@michalkleiner It happened on initial load after installing the module.

**Comment by matthewrayner** [comment]
This is a legacy site still using PHP 7.1.33 and Silverstripe 4.2.1 could that be something to do with it?

**Comment by matthewrayner** [comment]
It's happening when building the core cache  `SilverStripe\Core\Config\CoreConfig\CoreConfigFactory->createCore()`

**Comment by brettt89** [comment]
Hmm, it shouldn't be required (only if you wanted to use the helper job) so is only recommended. Is there any further stack trace / details you could provide?

**Comment by matthewrayner** [comment]
![image](https://user-images.githubusercontent.com/30171253/128461368-b4d1b7dc-e017-4b2b-b045-7872b7406002.png)

**Comment by brettt89** [comment]
This should now be resolved and released in 1.1.0

**Comment by lozcalver** [comment]
Looks like this has regressed, on a `dev/build` without the queuedjobs module installed:

>  Fatal error: Uncaught Error: Class "Symbiote\QueuedJobs\Services\AbstractQueuedJob" not found in /var/www/html/vendor/brettt89/silverstripe-garbage-collector/src/Jobs/RecurringAllGarbageCollectorJob.php:25 Stack trace: #0 /var/www/html/vendor/composer/ClassLoader.php(576): include() #1 /var/www/html/vendor/composer/ClassLoader.php(427): Composer\Autoload\{closure}() #2 [internal function]: ...

Class is just missing:

```php
if (!class_exists(AbstractQueuedJob::class)) {
    return;
}
```
